### ISSUE TITLE
async/await

### ISSUE THREAD

**Issue by RobLoach** [jstransformers/jstransformer#191]
Add docs on how to use this with async/await.

**Comment by calebeby** [comment]
:question: Confused what you are asking for :confused:

Do we have to change our implementations at all for JSTransformers to be compatible with `async`/`await`?

**Comment by tunnckoCore** [comment]
@calebeby no. @RobLoach is talking about that we should make a note for users for the `*Async` methods that they can use it with combination with `async/await` syntax.

```js
var md = require('jstrasformer')(require('jstransformer-marked')

async function myAsyncFn () {
  await md.renderAsync('some md')
}
```

**Comment by calebeby** [comment]
Ah. Ok
